.github/workflows/python-build-test.yml

```diff
@@ -0,0 +1,37 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest mypy black
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with flake8
+        run: |
+          flake8 src/pyprojroot tests
+      - name: MyPy (type) checking
+        run: |
+          mypy --strict src/pyprojroot
+      - name: Format with black
+        run: |
+          black --check --diff src/pyprojroot tests
+      - name: Test with pytest
+        run: |
+          PYTHONPATH=src pytest
```

Comment:
The correct thing to do is to install the package as editable:

`python -m pip install -e .`